Loader/Loader.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * File containing the Loader class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\Loader;
+
+/**
+ * JS and CSS loader.
+ */
+interface Loader
+{
+    /**
+     * Returns combined JS or CSS files.
+     *
+     * @param array $files File name array
+     *
+     * @return string Combined files
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException
+     */
+    public function getContent(array $files);
+
+    /**
+     * Returns content type.
+     *
+     * @param array $files File name array
+     *
+     * @return string
+     * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException
+     */
+    public function getContentType(array $files);
```

Comment:
Like above, mainly to avoid using the domain names from repository, something like `getCombinedFileContentType`, or `getMediaType` or ..